bamboo-specs/bamboo.yaml

```diff
@@ -255,18 +228,130 @@ version: 2
 plan:
   key: ORCA-OI
 plan-permissions:
-- users:
-  - bhazuka
-  - andrew.dorn
-  - rizbi.hassan
-  - scott.saxon
+- groups:
+  - bamboo-orca-devs
+  permissions:
+  - view
+  - edit
+  - build
+  - clone
+  - admin
+  - viewConfiguration
+...
+
+# ------------------------------------------------------------------------------------------------
+#bamboo plan for doing an ORCA documentation release
+---
+version: 2
+plan:
+  project-key: ORCA
+  key: ROD
+  name: Release ORCA Documentation
+stages:
+- Release Stage:
+    manual: true
+    final: true
+    jobs:
+    - Release ORCA Documentation
+
+Release ORCA Documentation:
```

Comment:
Comment on why we aren't using the `image` property to set up the container.

Reply:
using the orca container raises authentication error during npm deploy. See ORCA-467 for details.